src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComInvokeAction.cs

```diff
@@ -48,7 +52,16 @@ internal sealed class SplatInvokeBinder : CallSiteBinder
     {
         internal static readonly SplatInvokeBinder s_instance = new SplatInvokeBinder();
 
+        // The only callsite using this type is also marked as unsafe
+        [RequiresUnreferencedCode(Binder.TrimmerWarning)]
```

Comment:
I wonder if we can do something better here. Maybe make the field `private` and add a property over top that we can mark as `RequiresUnreferencedCode`. And then just suppress the warning in the static ctor. See https://github.com/dotnet/runtime/pull/49682#discussion_r596433523 for a similar comment there.

Reply:
Yes, that's a good idea, I think rather than suppressing I can also mark ctor a private as well and only mark property getter as RUC which would be the only entry point